src/test/java/org/broadinstitute/hellbender/utils/config/ConfigMarbleRollIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import htsjdk.samtools.ValidationStringency;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.tools.PrintReads;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.Collections;
+
+/**
+ * Integration test suite for Configuration tests.
+ * Created by jonn on 10/4/17.
+ */
+public class ConfigMarbleRollIntegrationTest extends CommandLineProgramTest  {
+
+    private static final String configFilePath = GATKBaseTest.gatkDirectory + "src/main/resources/org/broadinstitute/hellbender/utils/config/GATKConfig.properties";
+
+    @Override
+    public String getTestedToolName() {
+        return PrintReads.class.getSimpleName();
+    }
+
+    @Test
+    public void testPrintReadsWithConfigFile() throws Exception {
+
+        final File tmpDir = createTempDir("testPrintReadsWithConfigFile");
+
+        final String inputFile = publicTestDir + "NA12878.chr17_69k_70k.dictFix.bam";
+        final String outputFile = tmpDir + File.separator + "out.bam";
```

Comment:
Just use `createTempFile()` for `outputFile`, don't need to use `createTempDir()`

Reply:
Fixed!